commit 927840bc3f7010944d167f85c5d899c9f1efd34c
Author: Klaus Aehlig <aehlig@google.com>
Date:   Tue Jul 9 10:40:52 2013 +0200

    In the crontab example, look for the correct binary
    
    When ganeti-master-cleaner was merged back into ganeti-cleaner in 46118ed2,
    the binary to test for should have also been updated. Do this now.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

diff --git a/doc/examples/ganeti.cron.in b/doc/examples/ganeti.cron.in
index 964af57..1112b21 100644
--- a/doc/examples/ganeti.cron.in
+++ b/doc/examples/ganeti.cron.in
@@ -4,7 +4,7 @@ PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin
 */5 * * * * root [ -x @SBINDIR@/ganeti-watcher ] && @SBINDIR@/ganeti-watcher
 
 # Clean job archive (at 01:45 AM)
-45 1 * * * @GNTMASTERUSER@ [ -x @SBINDIR@/ganeti-master-cleaner ] && @SBINDIR@/ganeti-cleaner master
+45 1 * * * @GNTMASTERUSER@ [ -x @SBINDIR@/ganeti-cleaner ] && @SBINDIR@/ganeti-cleaner master
 
 # Clean job archive (at 02:45 AM)
 45 2 * * * @GNTNODEDUSER@ [ -x @SBINDIR@/ganeti-cleaner ] && @SBINDIR@/ganeti-cleaner node
